README.md

```diff
@@ -90,7 +90,17 @@ python -m pip install macrobond-data-api
 
 Macrobond Data API for Python officially supports Python 3.6+.
 
-## Using of system keyring
+## Using of system keyring for http proxy
+
+If your using a http proxy is recommended to use the system keyring.
```

Comment:
For users operating behind an HTTP proxy, it is advisable to utilize the system keyring. 
When employing WebClient, the system keyring is also recommended. 
This can be conveniently accomplished by executing the included script with the following command:

```console
python -c "from macrobond_data_api.util import *; save_proxy_to_keyring()"
```